frontend/app/shared/models/assessment.model.ts

```diff
@@ -81,3 +81,9 @@ export interface IAssessmentList {
   incomplete_results: boolean;
   items: Partial<Assessment>[];
 }
+
+export interface ITerritory {
```

Comment:
Je voie que tu as suivi la convention visant à ajouter un "I" majuscule devant le nom d'une interface. C'est très bien car c'est ce que l'on a fait dans le reste du projet. Pour info, il y a plusieurs possibilités pour nommer les interfaces afin de les distinguer des classes qui les utilisent. Voir : https://nicotsou.com/tltr-typescript-interfaces/

Tu peux fermer ce commentaire après lecture...

Reply:
ah merci pour l'article, tu préfères utiliser juste un "I" ou un "Is" ?

Reply:
"I" tout simplement. C'est la façon de faire qui me semble la plus courante.